src/main/scala/com/guidewire/cda/TableReader.scala

```diff
@@ -464,16 +466,42 @@ class TableReader(clientConfig: ClientConfig) {
    */
   private[cda] def fetchDataFrameForTableTimestampSubfolder(tableTimestampSubfolderInfo: TableS3LocationWithTimestampInfo): DataFrameWrapper = {
     val s3URI = tableTimestampSubfolderInfo.timestampSubfolderURI
-    val s3aURL = s"s3a://${s3URI.getBucket}/${s3URI.getKey}*"
+    val s3aURL = s"s3a://${s3URI.getBucket}/${s3URI.getKey}*.parquet"
     log.info(s"Reading '${tableTimestampSubfolderInfo.tableName}' from $s3aURL, on thread ${Thread.currentThread()}")
-    // Adding in column renaming capabilities - we ran across the column name "interval", and cannot get the jdbc driver to stop creating the column with
-    //   double quotes around it in the table definition. When we execute the Merged code, it thinks the value is a literal and won't insert.
-    //   Adding additional .withColumnRenamed() lines below works in the event multiple columns need to be renamed.
     val dataFrame = spark.sqlContext.read.parquet(s3aURL)
-      .withColumnRenamed("interval", "interval_")
+
+    val dataFrameWrapper = manageDataFrameColumns(dataFrame, tableTimestampSubfolderInfo)
+    dataFrameWrapper
+  }
+
+  /** Function that takes a DataFrame corresponding to some table and manages the addition and removal of
```

Comment:
This documentation does not mention dropping of `spatial`/`textdata` fields. Are these internal fields, or just ones that the database cannot handle?